fast64_internal/f3d/f3d_gbi.py

```diff
@@ -2807,8 +2975,15 @@ def get_f3d_mat_from_version(material: bpy.types.Material):
 
 
 class FMaterial:
-    def __init__(self, name, DLFormat):
+    def __init__(self, name, DLFormat, inline = False):
         self.material = GfxList("mat_" + name, GfxListTag.Material, DLFormat)
+        # when not inline, appending to the textures list will append to the material list due
+        # to the list being mutable. separating the lists allow individual textures to be bled
+        if inline:
+            self.textures = [GfxList(f"tex_{i}_" + name, GfxListTag.Material, DLFormat.Static) for i in range(2)]
+        else:
+            self.textures = [self.material] * 2
```

Comment:
In this case, do you think self.textureDLs should be something like:
```python
        if inline:
            self.textureDLs = [GfxList(f"tex_{i}_" + name, GfxListTag.Material, DLFormat.Static) for i in range(2)]
        else:
            self.textureDLs = None
```
I feel like even if this intent is more clear, functionally it would be better to leave the mutability case up as the correct behavior would execute even when the getter function is not used.